runtime/runtime-params-estimator/src/main.rs

```diff
@@ -73,8 +73,8 @@ struct CliArgs {
     #[clap(long)]
     drop_os_cache: bool,
     /// Print extra debug information
-    #[clap(long, multiple_occurrences = true, possible_values=&["io", "rocksdb", "least-squares"])]
-    debug: Vec<String>,
+    #[clap(long)]
+    debug: bool,
```

Comment:
as you're changing the argument values -- do we have any scripts that are executing this binary?